master/networking/mtu.md

```diff
@@ -1,130 +1,100 @@
+| 9001 (AWS Jumbo)       | 9001                  | 8981                                       | 8951                                    |
+| 1450 (OpenStack VXLAN) | 1450                  | 1430                                       | 1400                                    |
 
-When using Kubernetes self-hosted manifests, the CNI plugin derives the MTU value
-from the `veth_mtu` field of the calico-config ConfigMap, and it is set to `1440`
-by default. On restart of the `{{site.nodecontainer}}` pods, any references to
-`__CNI_MTU__` are replaced by the `veth_mtu` value and inserted into the CNI
-configuration file (aka conflist) at the directory specified by Kubernetes
-(currently defaults to `/etc/cni/net.d/`).
+**Recommended MTU for overlay networking**
 
-Restarting the `{{site.nodecontainer}}` pods will also update any {{site.prodname}}
-tunnel network interfaces on that node. From this point forward, any pods
-started will also have the updated MTU value.
+The extra overlay header used in IP in IP and VXLAN protocols, reduces the minimum MTU by the size of the header. (IP in IP uses a 20-byte header, and VXLAN uses a 50-byte header). Therefore, we recommend the following:
 
-### Setting tunnel MTU with a Felix environment variable
+- If you use VXLAN anywhere in your pod network, configure MTU size as “physical network MTU size minus 50”. 
+- If you use only IP in IP, configure MTU size as “physical network MTU size minus 20”
+- Set the workload endpoint MTU and the tunnel MTUs to the same value (so all paths have the same MTU)
 
-Passing in the environment variable `FELIX_IPINIPMTU` (or `FELIX_VXLANMTU`) when running the
-`{{site.nodecontainer}}` container will set the MTU for Felix to use.
+**MTU for flannel networking**
 
-When using the Kubernetes self-hosted manifests, Felix derives the IP-in-IP value from
-the `veth_mtu` field of the calico-config ConfigMap, which is set to `1440` by default.
+When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface. If using flannel with VXLAN, use the “{{site.prodname}} MTU with VXLAN” column in the table above for common sizes. 
 
-### Setting tunnel MTU with calicoctl
+#### Configure MTU for workloads
+  
+When you set the MTU, it applies to new workloads. To apply MTU changes to existing workloads, you must restart calico nodes. Restarting the calico/node pods takes values from the ConfigMap and starts rolling updates for any {{site.prodname}} tunnel network interfaces on the node. 
 
-To set the IP-in-IP MTU value for all {{site.prodname}} nodes in your cluster, use the
-following command to retrieve the current Felix settings.
+Edit the `calico-config` ConfigMap to set values in FelixConfiguration. For example:
 
-```bash
-calicoctl get felixconfig default --export -o yaml > felix.yaml
+```
+kubectl patch configmap/{{site.prodname}}-config -n kube-system --type merge \
```

Comment:
@bmckercher123 @lmm FYI, there's a bug here; the config map really is called `calico-config` not `Calico-config` or `Calico Enterprise-config`